main.nf

```diff
@@ -118,6 +123,16 @@ if (params.rlocation){
     nxtflow_libs.mkdirs()
 }
 
+// Custom trimming options
+params.length = 18
+params.clip_R1 = 0
+params.three_prime_clip_R1 = 0
+
+// Define regular variables so that they can be overwritten
+length = params.length
+clip_R1 = params.clip_R1
+three_prime_clip_R1 = params.three_prime_clip_R1
```

Comment:
No need to do this here if you don't have presets for different library prep kits.